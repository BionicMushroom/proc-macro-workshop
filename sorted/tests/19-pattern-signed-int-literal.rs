#[sorted::check]
fn main() {
    let x = 15i128;

    #[sorted]
    match x {
        0 => println!("0"),
        1 => println!("1"),
        -170141183460469231731687303715884105728 => {
            println!("-170141183460469231731687303715884105728")
        }
        _ => println!("other"),
    }
}
