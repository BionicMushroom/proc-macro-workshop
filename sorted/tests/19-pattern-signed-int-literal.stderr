error: -170141183460469231731687303715884105728 should sort before 0
 --> tests/19-pattern-signed-int-literal.rs:9:9
  |
9 |         -170141183460469231731687303715884105728 => {
  |         ^
